content/blog/2024-12-dev-update/index.md

```diff
@@ -0,0 +1,70 @@
+- Join the monthly virtual community meetings [here](https://groups.google.com/g/tlaplus/c/CpAEnrf-DHQ/m/YrORpIfSBwAJ)!
+- Start hacking on the tools themselves [here](https://github.com/tlaplus/tlaplus)!
+
+Let's start with some interesting non-coding-related developments & announcements:
+- [Leslie Lamport](https://lamport.azurewebsites.net/) published a new book, titled *A Science of Concurrent Programs*!
+  It's [freely available for download](https://lamport.azurewebsites.net/tla/science-book.html) but a physical edition is in the works.
+  I've been reading & enjoying it and asking lots of questions on the mailing list.
+  It's very much focused on the "TLA" part of TLA⁺ (meaning the Temporal Logic of Actions) so it's a good read if you want to go really in-depth on the logical theory underpinning the TLA⁺ language.
+- The [TLA⁺ Community Event 2025](https://conf.tlapl.us/2025-etaps/) was announced, to be co-located with [ETAPS 2025](https://etaps.org/2025/) in Hamilton, Ontario, Canada on May 4th, 2025.
+  We hope to see you there!
+  The talk proposal submission deadline is February 7th, 2025.
+- East of the Atlantic, [ABZ 2025](https://abz-conf.org/site/2025/) - the 11th International Conference on Rigorous State-Based Methods - will take place from June 10th to 13th of 2025 in Düsseldorf, Germany.
+  TLA⁺-related papers are welcome; the deadline for submitting abstracts is February 3rd, 2025.
+- [Jack Vanlighty](https://jack-vanlightly.com) wrote [a blog post](https://jack-vanlightly.com/blog/2024/11/19/obtaining-statistical-properties-through-modeling-and-simulation) on using TLA⁺ to obtain statistical properties of distributed system designs.
+- [Lorin Hochstein](http://lorinhochstein.org/) has been on a TLA⁺ blogging *tear*, publishing three excellent blog posts in quick succession: [*Specifying serializability in TLA⁺*](https://surfingcomplexity.blog/2024/10/28/serializability-and-tla/), [*Multi-version concurrency control in TLA⁺*](https://surfingcomplexity.blog/2024/10/31/multi-version-concurrency-control-in-tla/), and [*Extending MVCC to be serializable, in TLA⁺*](https://surfingcomplexity.blog/2024/11/03/extending-mvcc-to-be-serializable-in-tla/).
+  The first post is especially effective as an introduction to using refinement to express whole-system safety properties that just can't be done with single-state formulas.
+- At the November virtual community meeting, [Feng Wenhan](https://github.com/fwhdzh) presented his work using the TLC state graph export function to drive model-based testing; some shortfalls in the state graph export functionality led to the proposal of designing a [robust state graph export format](https://github.com/tlaplus/tlaplus/issues/1073).
+- Also at the November virtual community meeting, Leslie Lamport announced his retirement at the end of the year at the age of 83.
+  A well-earned rest, although his contributions will be missed!
+
+Now on to coding-related community developments:
```

Comment:
I know you pinged me about it but have you confirmed that the other individuals listed here consider this an accurate representation of their work? Additionally, is this list comprehensive? The same question applies to the previous section. It's important to ensure that we are inclusive and avoid inadvertently excluding any part of our community.

Reply:
For example, Julia's work on the TLA+ Foundation website does not appear in this newsletter.

Reply:
That is a good one to add! I did not think to check the foundation repo itself.

Reply:
I added a note about Julia's work.